plugins/woocommerce-blocks/assets/js/atomic/blocks/product-elements/add-to-cart-form/style.scss

```diff
@@ -23,3 +24,7 @@
 		}
 	}
 }
+
+.woocommerce div.product .wc-block-add-to-cart-form form.cart button.single_add_to_cart_button {
```

Comment:
With the Twenty Twenty-Four theme at least, it seems like this just adds some unnecessary margin that further increases the gap between the button and the elements below it.

I'm also weary of the margin being added here with such an aggressive selector because I feel like it has the potential to cause problems with themes.

What do you think?

Reply:
Yeah I explained this in my PR. I don't know how else to do this other than we add styles for each themes.

Reply:
I tested with Tsubaki theme. I guess I should have tested a few others.

Reply:
I was more asking whether or not the button _needs_ a margin. The issue was with the quantity stacking, right?

Reply:
Right, but if you don't add the margin to the add to cart button, you will see a staggered effect.

Reply:
> Right, but if you don't add the margin to the add to cart button, you will see a staggered effect.

Sorry, what do you mean by staggered effect?

Reply:
I see, it just doesn't work at all without it 😕

Reply:
![file.png](https://github.com/woocommerce/woocommerce/assets/2132595/13b01b41-ff00-4e69-8ac7-4b5f57d8e39f)

![file.png](https://github.com/woocommerce/woocommerce/assets/2132595/a4c5a42d-bf1e-4d20-a0f4-830370abd5e1)